elastic/apm-server
# Fix genpackage ilm_policy name for rollup interval indices

## Motivation/summary



Fix ILM policy name in rollup interval index template. e.g. change expected ILM policy name from `metrics-apm.service_destination_interval_metrics-default_policy.1m` to `metrics-apm.service_destination_1m_metrics-default_policy`.



## Checklist




## How to test these changes



- Send events so that backing index is created using index template
- Inspect the index in stack monitoring to confirm ILM policy exists.

## Related issues



Closes https://github.com/elastic/apm-server/issues/10817
